pkg/ddl/000026_add_commit_version.up.sql

```diff
@@ -0,0 +1,4 @@
+BEGIN;
+ALTER TABLE graveler_commits
+    ADD COLUMN IF NOT EXISTS version text DEFAULT 'commit:v1' NOT NULL;
```

Comment:
make numerical.
find new name for column

Reply:
If I make it numerical we'll lose compatibility with what we had so far which was 'commit:v1'

Reply:
And what's wrong with `version`?

Reply:
If it's a string. please document it clearly as "must have sortable keys" or something.  Otherwise how do we know whether we're before or after a given version?
Also, this is a pretty large field on disk (doesn't matter that much for commits).
Personally I'd go for named (boolean) feature toggles, or ints.

Reply:
Combine multiple inputs from the above and @talSofer - `version` will be numeric where zero is the initial version and each change will have a new number.
In order to identify it in the code, we will have something like an enum where each value will be a change we made.
Version will be part of the commit identify will be savedd/loaded

Reply:
Commits hash for version 0 were calculated with 'commit:v1'. Not sure we need to keep it compatible, as I can't think of a scenario where we recalculate it.